movement/movement.h

```diff
@@ -270,6 +270,10 @@ typedef struct {
 
     // low energy mode countdown
     int32_t le_mode_ticks;
+    uint8_t debounce_ticks_light;
+    uint8_t debounce_ticks_alarm;
+    uint8_t debounce_ticks_mode;
+    bool ignore_alarm_btn_after_sleep;
```

Comment:
This variable has the potential to be eliminated.

Reply:
Which variable specidfically?

Reply:
`ignore_alarm_btn_after_sleep`